app/javascript/src/learning_hours.js

```diff
@@ -0,0 +1,5 @@
+import { initializeDataTable } from '../datatable'
+
+$(() => {
+  initializeDataTable('table#learning-hours')
```

Comment:
Similarly here — do we have to check for the existence of `table#learning-hours` before trying to initialize the data table?